osu.Game/Graphics/UserInterface/OsuTabControl.cs

```diff
@@ -22,6 +22,22 @@ namespace osu.Game.Graphics.UserInterface
 {
     public class OsuTabControl<T> : TabControl<T>
     {
+        protected readonly Bindable<Color4> AccentColourBindable = new Bindable<Color4>();
+
+        public Color4 AccentColour
+        {
+            get => AccentColourBindable.Value;
+            set
+            {
+                AccentColourBindable.Value = value;
+
+                if (Dropdown is IHasAccentColour dropdown)
+                    dropdown.AccentColour = value;
+                foreach (var i in TabContainer.Children.OfType<IHasAccentColour>())
+                    i.AccentColour = value;
+            }
+        }
```

Comment:
This is half-good, half-not-really now. Bindable is a plus, but what's the point if you end up propagating the value down instead anyways? This could be much nicer if `IHasAccentColour.AccentColour` was an `IBindable<Color4>`.